src/Symfony/Component/Serializer/Encoder/YamlEncoder.php

```diff
@@ -28,9 +28,17 @@ class YamlEncoder implements EncoderInterface, DecoderInterface
 
     public const PRESERVE_EMPTY_OBJECTS = 'preserve_empty_objects';
 
+    public const YAML_INLINE = 'yaml_inline';
+    public const YAML_INDENT = 'yaml_indent';
+    public const YAML_FLAGS = 'yaml_flags';
+
     private $dumper;
     private $parser;
-    private $defaultContext = ['yaml_inline' => 0, 'yaml_indent' => 0, 'yaml_flags' => 0];
+    private $defaultContext = [
```

Comment:
There are other places in this class where the new constants could be used as well

Reply:
True. We should at least use them within this class.

Reply:
Done, both here and in the related tests.